project/Dependencies.scala

```diff
@@ -149,6 +150,7 @@ object Dependency {
     val Mockito = "1.10.19"
     val ScalaCheck = "1.13.5"
     val ScalaTest = "3.0.4"
+    val UsiTestUtil = "0.1.4-a97fc25-SNAPSHOT"
```

Comment:
did we not merge yet? I would like to add a rule for snapshot packages in USI to expire after a period (30 days?), we should not avoid using them in PRs and just use them for testing changes and coding

Reply:
> we should not avoid using them in PRs and just use them for testing changes and coding

The is no USI master package with the required changes yet. I'm not sure I understand your comment. The snapshot dependency is not supposed to land on Marathon master.

Reply:
Ok, should we mark this as a WIP PR then? We do use snapshot packages for precisely the reason you outline

Reply:
The dependency is already `0.1.7`. I'm not sure what you mean.